draft-ietf-tls-esni.md

```diff
@@ -777,9 +788,13 @@ ClientHelloOuter as follows:
 - `enc` is replaced with the empty string.
 - `payload` is replaced with the value computed above.
 
-If the client offered ECH in the first ClientHello, then it MUST offer ECH in
-the second. Likewise, if the client did not offer ECH in the first ClientHello,
-then it MUST NOT not offer ECH in the second.
+#### Rejected ECH
+
+If the client-facing server sent HelloRetryRequestOuter without the
+"encrypted_client_hello" extension, then the client presumes rejection and
+constructs the second ClientHelloOuter as described in {{RFC8446}}. In
+particular, it MUST copy the "encrypted_client_hello" from the first
+ClientHelloOuter into the second.
```

Comment:
"copy" seems like a mistake here. As I understand it, this will be undecryptable anyway because of the AAD, so I would prefer random content.

Reply:
AFAIK this behavior is mandated by 8446. Perhaps we should add an OPEN ISSUE Here and bring it to the list?

Reply:
yeah.

Reply:
Added a TODO.